tests/python/contrib/test_ethosu/test_codegen.py

```diff
@@ -1564,6 +1564,30 @@ def fully_connected(x):
     )
 
 
+@pytest.mark.parametrize("accel_type", ["ethos-u55-256", "ethos-u65-256"])
+@pytest.mark.parametrize("ifm_shape", [(1, 16)])
```

Comment:
Does it work if the first dimension of IFM is not 1? If yes, we should test it, if not, we should make it explicit in the `is_valid` checks.

Reply:
It works because width and channels dimensions are used for calculations, test cases are added.